lib/foreman_maintain/concerns/pulp_common.rb

```diff
@@ -1,7 +1,7 @@
 module ForemanMaintain
   module Concerns
     module PulpCommon
-      def data_dir
+      def pulp_data_dir
```

Comment:
I changed this to pulp_data_dir as this was being picked up while doing db backup of pulp2/pulpcore